src/careamics/lvae_training/dataset/types.py

```diff
@@ -15,6 +15,7 @@ class DataType(Enum):
     OptiMEM100_014 = 10
     SeparateTiffData = 11
     BioSR_MRC = 12
+    MultiCropDset = 13  # for the case when we have a set of differently sized crops for each channel.
```

Comment:
Just have a doubt: is this actually a `DataType` or a new `Dataset` class (e.g., like `MultiFileDset`)? The naming is highly misleading ...